### ISSUE TITLE
Implement serializer for Offset

### ISSUE THREAD

**Issue by Tolyandre** [AndyBeverlySchool/MongoDb.Bson.NodaTime#10]
Hey, I am working on a project that uses this library. The library actually is great. Are there any plans to implement a serializer for [Offset](https://nodatime.org/3.0.x/api/NodaTime.Offset.html)?

I checked using this code, it currently produces an empty object:
```c#
using System;
using MongoDB.Bson;
using MongoDb.Bson.NodaTime;
using NodaTime;

namespace TestOffset
{
    class Data
    {
        public Offset Offset { get; set; }
    }

    class Program
    {
        static void Main(string[] args)
        {
            NodaTimeSerializers.Register();

            var data = new Data
            {
                Offset = Offset.FromHours(4),
            };

            Console.WriteLine(data.ToBsonDocument()); // { "Offset" : { } }
        }
    }
}
```

**Comment by tetious** [comment]
Hi! I'm glad it is working well for you.

Adding support for Offset should be fairly straightforward, as there is a serialization pattern for it in NodaTime.Text. (https://nodatime.org/3.0.x/api/NodaTime.Text.OffsetPattern.html)

We already have a base class that wraps things nicely, and using that might make it Just Work(TM). See DurationSerializer.cs for an example. 

I'd happily accept a PR if you want to take a stab at it. :)

**Comment by tetious** [comment]
Thanks for the PR and the build fix! This is pushed to nuget as 2.1.0.
